frontend/src/i18n/locales/de/notifications.json

```diff
@@ -0,0 +1,14 @@
+{
+  "You don't have any notifications right now": "Sie haben im Moment keine Benachrichtigungen",
+  "No message": "Keine Nachricht",
+  "Mark as read": "Als gelesen markieren",
+  "Show notifications": "Benachrichtigungen anzeigen",
```

Comment:
Meldungen anzeigen

Reply:
Ok, thanks. I changed it to "Nachrichten".